frontend/src/components/loading/index.jsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Flag, { FlagGroup } from '@atlaskit/flag';
+import Spinner from '@atlaskit/spinner';
+
+function Loading({ loading }) {
+  const flags = [];
+
+  if (loading) {
+    const loadingIcon = () => (
+      <div style={{ width: 16, height: 16 }}>
```

Comment:
Does the Spinner component have a default style size that could be leveraged in lieu of the style magic numbers?

Reply:
No, this is a layout hack so it can replace an icon in the Flag component, which I was following from the library's example page.